Cargo.toml

```diff
@@ -23,6 +23,7 @@ libudev = { version = "0.3.0", optional = true }
 unescaper = "0.1.3"
 
 [target.'cfg(any(target_os = "ios", target_os = "macos"))'.dependencies]
+core-foundation = "0.10.0"
 core-foundation-sys = "0.8.4"
```

Comment:
Can we remove this one, then?

Reply:
Unfortunately not as they are used with IOKit and there are neither exports from core-foundation nor io-kit-sys. Made the imports in question explicit at least.